### PR TITLE
Remove test of Jakarta XML Registries

### PR DESCRIPTION
**Pull request by cesarhernandezgt**
Tackle removal task described in #209

### CODE REVIEW

File: install/connector/bin/server_policy.append [hunk 1]
```diff
@@ -87,4 +87,4 @@ grant {
     permission java.security.SecurityPermission  "CTSPermission1_name";
     permission java.security.SecurityPermission  "CTSPermission1_name2";
     permission javax.xml.ws.WebServicePermission "ConnectorPermission1_name2";
```
Comment (scottmarlow) [inline install/connector/bin/server_policy.append:89]: I don't see a change for JAXR on this file.

File: install/jakartaee/bin/sig-test-pkg-list.txt [hunk 2]
```diff
@@ -157,7 +157,7 @@ javax.xml.bind.attachment
 javax.xml.bind.helpers
 javax.xml.bind.util
 javax.xml.registry
```
Comment (scottmarlow) [inline install/jakartaee/bin/sig-test-pkg-list.txt:159]: I think that javax.xml.registry is the package for JAXR and should be removed.
Also javax.xml.registry.infomodel as well.

File: install/jakartaee/bin/sig-test-pkg-list_se9.txt [hunk 3]
```diff
@@ -174,7 +174,7 @@ javax.xml.bind.attachment
 javax.xml.bind.helpers
 javax.xml.bind.util
 javax.xml.registry
```
Comment (scottmarlow) [inline install/jakartaee/bin/sig-test-pkg-list_se9.txt:176]: same for javax.xml.registry here

File: install/jaxr/bin/sig-test-pkg-list.txt [hunk 4]
```diff
@@ -23,5 +23,5 @@
 ##
```
Comment (scottmarlow) [inline install/jaxr/bin/sig-test-pkg-list.txt:24]: also the javax.xml.registry line

File: install/jaxr/docs/JAXRSpecAssertions.html [hunk 5]
```diff
@@ -843,7 +843,7 @@ <h2> Jakarta Registries 1.0 - 1.0<br>
 </font></TD><TD valign="center" align="center"><font size="1PT">true</font></TD><TD valign="center" align="center"><font size="1PT">technology</font></TD><TD valign="center" align="center"><font size="1PT">active</font></TD><TD valign="center" align="center"><font size="1PT">false</font></TD>
 </TR>
```
Comment (scottmarlow) [inline install/jaxr/docs/JAXRSpecAssertions.html:844]: I think removing install/jaxr is expected, I've never pruned a TCK test before but that is what I think is right.  Good that we have others that know for sure.

File: internal/docs/jaxr/CtsJAXRTestPlan.html [hunk 6]
```diff
@@ -200,55 +200,55 @@ <h2>
 <p><b>Explicit and Implicit Testing</b>
```
Comment (scottmarlow) [inline internal/docs/jaxr/CtsJAXRTestPlan.html:200]: IMO probably should also remove internal/docs/jaxr folder.

File: src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_default.xml [hunk 7]
```diff
@@ -40,4 +40,4 @@
 			</writer>
 		</chunk>
 	</step>
-</job>
```
Comment (scottmarlow) [inline src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_default.xml:43]: no jaxr changes made here, looks like an unexpected end of file marker change.

File: src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_multiple_steps-customCA.xml [hunk 8]
```diff
@@ -81,4 +81,4 @@
 		    </checkpoint-algorithm>
```
Comment (scottmarlow) [inline src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_multiple_steps-customCA.xml:81]: revert this file and other changes that only change end of file marker.

File: src/com/sun/ts/tests/internal/implementation/sjsas/jaxr/ee/javax_xml_registry/RegistryServicects/JAXRClient.java [hunk 9]
```diff
@@ -34,8 +34,8 @@
 import java.rmi.*;
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/internal/implementation/sjsas/jaxr/ee/javax_xml_registry/RegistryServicects/JAXRClient.java:34]: There are no jakarta.xml.registry classes, I think that JAXRClient needs to be removed.

I did "ag --ignore .git  -l javax.xml.registry" locally to find the files that reference JAXR, which includes most of the files that you updated.  The JAXRClient is only used by the src/com/sun/ts/tests/jaxr/ files which I think you will delete.

**Comment by scottmarlow** [comment]
Also, we should modify src/com/sun/ts/lib/harness/keyword.properties and remove lines:
```
com/sun/ts/tests/jaxr = jaxr jaxr_web_profile javaee_optional javaee_web_profile_optional
com/sun/ts/tests/signaturetest/jaxr = signaturetest jaxr
```

**Comment by cesarhernandezgt** [comment]
In 832f18ed6fb010179d9e644d2089cbf97fdefd28 I basically did:

rm -rf src/com/sun/ts/tests/jaxr/api/javax_xml_registry/ and
rm -rf src/com/sun/ts/tests/jaxr/api/javax_xml_registry_infomodel/
rm -rf  src/com/sun/ts/tests/jaxr/ee/javax_xml_registry/
rm -rf src/com/sun/ts/tests/jaxr/ee/javax_xml_registry_infomodel
rm -rf src/com/sun/ts/tests/internal/implementation/sjsas/jaxr/ee/javax_xml_registry/

**Comment by cesarhernandezgt** [comment]
Thanks for the review and pointers @scottmarlow I applied 99% of the suggestions here: e3d033e6a09818623d13aeb7050a5ddb8b524e6e

Now I think I have ~600 files that were affected by the javax.xml.registry and javax.xml.registry.infomodel initial commit that migrated to Jakarta namespace that I need to evaluate if the change proceed or we need to actually delete.

At this point I need to look back at my initial email thread because somehow I just identified that after the deletion of the folders in 832f18e I would be ended up with jus 18 files to analyze:

```
This will borrow the files under the test package to 18:

install/connector/bin/server_policy.append
install/jakartaee/bin/sig-test-pkg-list.txt
install/jakartaee/bin/sig-test-pkg-list_se9.txt
install/jaxr/bin/sig-test-pkg-list.txt
install/jaxr/docs/JAXRJavadocAssertions.html
install/jaxr/docs/JAXRSpecAssertions.html
internal/docs/jaxr/CtsJAXRTestPlan.html
internal/docs/jaxr/JAXRJavadocAssertions.xml
internal/docs/jaxr/JAXRSpecAssertions.xml
src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_default.xml
src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_multiple_steps-customCA.xml
src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_multiple_steps.xml
src/com/sun/ts/tests/jaxr/common/JAXRCommonClient.java
src/com/sun/ts/tests/jaxr/common/JAXR_Util.java
src/com/sun/ts/tests/servlet/api/javax_servlet/servletcontext305/javax.servlet.ServletContainerInitializer
src/com/sun/ts/tests/signaturetest/signature-repository/javax.xml.registry.sig_1.0_se8
user_guides/jakartaee/src/main/jbake/content/rules.adoc
user_guides/jaxr/src/main/jbake/content/packages.inc
```

I'll keep this as a Draft PR since I don't have experience running TCK scope but I'm happy to keep learning and helping.

File: install/jaxr/bin/sig-test-pkg-list.txt [hunk 10]
```diff
@@ -22,6 +22,4 @@
 # parent package's signature is checked.
 ##
 
-javax.xml.registry
```
Comment (scottmarlow) [inline install/jaxr/bin/sig-test-pkg-list.txt:25]: I think that the install/jaxr folder can be deleted, we probably can look for references to that folder, some of which will be from ant scripts, that need to be partially updated to not reference install/jaxr or jax stuff indirectly.  It is hard to search for these, as jaxr matches a few other longer technology names.

**Comment by scottmarlow** [comment]
I'll continue the review soon, I don't trust what I see on github.

**Comment by alwin-joseph** [comment]
Is there any need to retain the src/com/sun/ts/tests/jaxr/common folder ? If that is required then we will have to use jakarta.xml.registry api jar to compile it. It is my opinion that we remove this folder too.

File: src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_default.xml (hunk 7 above)
Comment (scottmarlow) [reply src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_default.xml]: I still see this unexpected change to this file.

**Comment by scottmarlow** [comment]
@cesarhernandezgt sorry, I just read your latest note...

> Thanks for the review and pointers @scottmarlow I applied 99% of the suggestions here: e3d033e
> 
> Now I think I have ~600 files that were affected by the javax.xml.registry and javax.xml.registry.infomodel initial commit that migrated to Jakarta namespace that I need to evaluate if the change proceed or we need to actually delete.
> 
> At this point I need to look back at my initial email thread because somehow I just identified that after the deletion of the folders in 832f18e I would be ended up with jus 18 files to analyze:
> 
> This will bring the files under the test package to 18:
> 
> install/connector/bin/server_policy.append
> install/jakartaee/bin/sig-test-pkg-list.txt
> install/jakartaee/bin/sig-test-pkg-list_se9.txt
> install/jaxr/bin/sig-test-pkg-list.txt
> install/jaxr/docs/JAXRJavadocAssertions.html
> install/jaxr/docs/JAXRSpecAssertions.html
> internal/docs/jaxr/CtsJAXRTestPlan.html
> internal/docs/jaxr/JAXRJavadocAssertions.xml
> internal/docs/jaxr/JAXRSpecAssertions.xml
> src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_default.xml
> src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_multiple_steps-customCA.xml
> src/com/ibm/jbatch/tck/testJobXml/META-INF/batch-jobs/job_chunk_globaltran_multiple_steps.xml
> src/com/sun/ts/tests/jaxr/common/JAXRCommonClient.java
> src/com/sun/ts/tests/jaxr/common/JAXR_Util.java
> src/com/sun/ts/tests/servlet/api/javax_servlet/servletcontext305/javax.servlet.ServletContainerInitializer
> src/com/sun/ts/tests/signaturetest/signature-repository/javax.xml.registry.sig_1.0_se8
> user_guides/jakartaee/src/main/jbake/content/rules.adoc
> user_guides/jaxr/src/main/jbake/content/packages.inc
> 
> I'll keep this as a Draft PR since I don't have experience running TCK scope but I'm happy to keep learning and helping.
> 
> 

Are you suggesting that we just start a new branch and copy in updated files?

**Comment by cesarhernandezgt** [comment]
@scottmarlow 
The issue I have with this PR,  is that I started with the javax. -> jakarta  approach to identify the files. That made it easier to see the candidate files for deletion.
I performed the deletion and base on your second review and also Alwin reviews it seems I still need to delete a couple of files.

After the deletion, if we still have files with the package migration applied,  we need to analyze if we rollback the changes on that files or if they are also candidate for deletion.

**Comment by scottmarlow** [comment]
I'm going to close this draft pr, so no one gets confused by it.

**Pull request closed without merging**